Cleanup github actions documentation (#106)
terramate-io/terramate-docs: cli/automation/github-actions/index.md

Please see the CLI-only examples in the [blueprints](#terramate-blueprints) section for automating Terramate CLI without Terramate Cloud in GitHub Actions.
:::

### Install the Terramate GitHub App

It's recommended to install the [Terramate GitHub App](../../../cloud/integrations/github.md) in your GitHub organization
when automating GitHub in GitHub Actions. The app provides well-formatted previews of your IaC changes inside of Pull Requests
that are easy to understand. You can install the app via the integrations section in your Terramate Cloud account,
for details please see the [GitHub integration](../../../cloud/integrations/github.md) documentation.

![GitHub App Pull Request Previews](../../assets/automation/pull-request-preview.png "GitHub App Pull Request Previews")

### Configure GitHub Trust in your Terramate Cloud Organization

> To allow Terramate CLI to sync data from GitHub Actions to Terramate Cloud, setting up OpenID Connect (OIDC) is required.
> For that, you must configure the [GitHub Trust](../../../cloud/organization/settings.md#general-settings) in the settings
> page of your Terramate Cloud Organization. This allows you to establish a trust relationship with GitHub Actions, which is
> required to sync data from GitHub, and specify whether to sync the entire organization or a specific repository with
> Terramate Cloud.

Reviewer: ```suggestion
To sync data from GitHub Actions to Terramate Cloud using the Terramate CLI, set up OpenID Connect (OIDC). Configure the [GitHub Trust](../../../cloud/organization/settings.md#general-settings) in your Terramate Cloud Organization settings. This establishes a trust relationship with GitHub Actions, allowing you to sync data from GitHub. You can sync the entire organization or a specific repository with Terramate Cloud.
```

Reviewer: optional suggestion..

- uses: hashicorp/setup-terraform@v3
  with:
    terraform_wrapper: false
```
:::

To install Terraform using the [`hashicorp/setup-terraform`](https://github.com/hashicorp/setup-terraform) GitHub Action, you must disable the included wrapper.

> The `terraform` wrapper script sets up outputs for follow-up GitHub Actions steps. The recommended method of using the [`terramate-io/terramate-action`](https://github.com/terramate-io/terramate-action) is through its supported wrapper, which shares outputs and exit codes for subsequent actions.

Reviewer 2: ```suggestion
The `terraform` wrapper script sets up outputs for follow-up GitHub Actions steps. The recommended method of using the [`terramate-io/terramate-action`](https://github.com/terramate-io/terramate-action) is through its own wrapper, which shares outputs and exit codes for subsequent actions.
```
I think it's better to understand this way but I could be wrong.
